docs/changelog/0023-remove-riak.md

```diff
@@ -0,0 +1,24 @@
+<!--THIS FILE IS AUTOGENERATED: DO NOT EDIT-->
```

Comment:
I think you (unfortunately) have to manually remove this file if this was the previously created one before a numbering change